### DISCUSSION TITLE
Day passes query

### DISCUSSION THREAD

**Discussion by benaunt** [Budibase/budibase#8675] [General]
Hi,

Could you please clarify if the advertised day passes are the total number of unique logged sessions per day or per month.

Eg for the lowest business tier plan of 500 day passes. Does it allow for 500 log ins every day in a month by different people or 500 total for the month noting that logging in multiple times a day only counts as one pass out of the 500.

Could you also clarify if any of the below counts as a day pass.

- Developer logging into the editor
- Developer previewing the app via the editor
- Developer running the published production app
- Viewing published apps in non production environments, once the budibase supports this
- When running multiple instances of budibase, to test product updates or in house devops. If users open the non production instance, will this count?

For the last dot point, when running multiple instances. Can they be attached to the same license or can a free licence be applied to ensure the instance has feature parity with the licensed production instance.

Thanks

**Comment by melohagan** [comment]
Hey @benaunt 

The advertised day passes are the total number of unique logged sessions per month. 
So in your example it would be: 
> 500 total for the month noting that logging in multiple times a day only counts as one pass out of the 500.

All bullet points outlined would initiate a day pass, but as you mentioned once active, no more day passes will be used for that particular user until a 24hr period has passed.

When running multiple instances, you would need a licence for each instance.

Hope this helps clarify things!
